mobile/src/bundles/common/components/header/header.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Text, View } from '~/bundles/common/components/components';
+import { TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/global-styles';
+
+import { styles } from './styles';
+
+const Header = (): JSX.Element => {
```

Comment:
This header should be a separate navigation component named OnboardingNavigation (for example) and have a Drawer navigation 
Investigate some variants of how you can customise the screen header in the Drawer navigator. See [this PR](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/83#pullrequestreview-1592867827) comments also to see how the navigation structure should look like

Reply:
While my research, I found out that we can pass another component as a header for Drawer Navigator, like this: 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/768c688f-764c-40e5-a288-39dc800fe39e)
So basically, I can use this created component by passing navigation as a props and by click on the icon drawer opens, like this (I will fix this ts errors): 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/d2e2fa0f-64e5-4e89-95df-ba0e01567a6d)

Is it a good approach?

Reply:
Yes, good